docs/src/pages/power-profiler.js

```diff
@@ -207,6 +223,68 @@ function PowerProfiler() {
                   </>
                 )}
               </div>
+              <div className="col col--4">
+                <div className="profilerInput">
+                  <label>Backlight</label>
+                  <input
+                    checked={backlightEnabled}
+                    id="backlight"
+                    {...bindBacklightEnabled}
+                    className="toggleInput"
+                    type="checkbox"
+                  />
+                  <label htmlFor="backlight" className="toggle">
+                    <div className="toggleThumb" />
+                  </label>
+                </div>
+                {backlightEnabled && (
+                  <>
+                    <div className="profilerInput">
+                      <label>LED Quantity</label>
+                      <div className="inputBox">
+                        <input {...bindBacklightQuantity} type="number" />
+                      </div>
+                    </div>
+                    <div className="profilerInput">
+                      <label>LED Voltage</label>
+                      <select {...bindBacklightColor}>
+                        {Object.entries(backlightLEDs).map(([c, v]) => (
+                          <option value={c}>
+                            {c} ({v.toFixed(1)} V)
+                          </option>
+                        ))}
+                        <option value="custom">Custom</option>
+                      </select>
+                      {backlightColor === "custom" && (
+                        <div className="profilerInput">
+                          <div className="inputBox">
+                            <input {...bindBacklightVoltage} type="number" />
+                            <span>V</span>
+                          </div>
+                        </div>
+                      )}
+                    </div>
+                    <div className="profilerInput">
+                      <label>LED Resistence</label>
```

Comment:
```suggestion
                      <label>LED Resistance</label>
```
Will probably want to update all other uses of resistance -> resistance